packages/slice-machine/lib/builders/SliceBuilder/SideBar/components/ImagePreview.tsx

```diff
@@ -78,14 +78,16 @@ const ImagePreview: React.FC<ImagePreviewProps> = ({
             {display ? (
               <Fragment>
                 <Flex sx={{ flexDirection: "column" }}>
-                  <Button
-                    sx={{ mb: 3 }}
-                    variant={preventScreenshot ? "disabled" : "primary"}
-                    disabled={preventScreenshot}
-                    onClick={onScreenshot}
-                  >
-                    Take screenshot
-                  </Button>
+                  {!preventScreenshot ? (
+                    <Button
+                      sx={{ mb: 3 }}
+                      variant="primary"
+                      disabled={preventScreenshot}
+                      onClick={onScreenshot}
```

Comment:
maybe we can remove this line as the component wont be rendered with disabled = true

Reply:
You mean the 'disabled' prop, right?